buf.gen.yaml

```diff
@@ -1,23 +1,29 @@
-#!/usr/bin/env -S buf generate buf.build/authzed/api:125e39e67bf60b2d19c4de15aaa69cb1bcbda28c --template
+#!/usr/bin/env -S go run github.com/bufbuild/buf/cmd/buf generate buf.build/authzed/api:125e39e67bf60b2d19c4de15aaa69cb1bcbda28c --template
```

Comment:
this makes me think there's something we could do (in the future) with the buf api. fetch the latest tag and generate off it or something.

Reply:
that'd be nice, something that updates the tag used, and then generates the code